src/components/Signup/Signup.scss

```diff
@@ -0,0 +1,116 @@
+.Signup {
+  display: flex;
+  padding: 44px 10px;
+  justify-content: space-around;
+}
+
+.Signup__info {
+  display: none;
+
+  h2 {
+    line-height: 1.1;
+  }
+}
+
+.Signup__info__text::before {
+  content: ' ';
+  position: absolute;
+  background: url('../../../assets/icon/icons8-double-tick.png') no-repeat;
+  top: 0;
+  left: 0;
+  width: 20px;
+  height: 20px;
+  background-size: 100%;
+}
+
+.Signup__form__header {
+  text-align: center;
+  font: {
+    size: 2em;
+    weight: 300;
+  }
+}
+
+.Signup__form {
+  width: 100%;
+  max-width: 400px;
+
+  .signup-error {
+    // border: 3px solid #dd2c00;
+    color: #dd2c00;
+    // background: #dd2c00;
+    // padding: 1rem 0.5rem;
```

Comment:
Why do we have these commented lines here? Please remove them.

Reply:
Thanks for pointing it out.